cmd/ipfs/init.go

```diff
@@ -142,6 +148,29 @@ func addTheWelcomeFile(conf *config.Config) error {
 	return nil
 }
 
+func initializeIpnsKeyspace(conf *config.Config) error {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
This operation could be done at the same time as `addTheWelcomeFile` so we dont have to instantiate a node two different times. If you want me to rewrite that in that way, just say the word.